### PR TITLE
Fix/selected execution mode

### LINKED ISSUE #833: 🐞 No execution mode selected when opening an existing agent
image

### PR DESCRIPTION
**Pull request by Zibi95**
fix #833 AgentForm reads actual execution mode and pass it to Select component

Animation

@udislau

### CODE REVIEW

**Review by udislau — CHANGES_REQUESTED** [review]
1) Go to existing agent with custom execution mode, select normal or interpreter. You can’t save it now


https://github.com/10clouds/aiconsole/assets/30052145/51683ce5-71c2-437d-b081-708926c3ceee



2) Try to replicate the video, after `restore original` execution mode field is empty.

https://github.com/10clouds/aiconsole/assets/30052145/16444c34-1b45-4e2a-8b09-b4802384b3cf

File: frontend/src/components/editables/assets/AgentForm.tsx [hunk 1]
```diff
@@ -1,30 +1,17 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
+import { useEffect, useMemo, useState } from 'react';
+
```
[the remark this answers is not in the document]
Comment (Zibi95) [reply frontend/src/components/editables/assets/AgentForm.tsx:3]: Do you mean that empty line? Ah, I did that reflexively. I've learned to separate imports from node_modules and local imports.

**Comment by nkvch** [comment]
Solved conflicts in #906

**Comment by nkvch** [comment]
Solved conflicts in #906

**Pull request closed without merging**